hack/fluent-receiver-template.yaml

```diff
@@ -101,7 +101,7 @@ objects:
             - name: fluentd
               command: ["fluentd","-c","/etc/fluent/configs.d/user/fluent.conf","--no-supervisor"] 
               image: ${IMAGE}
-              imagePullPolicy: Always
+              imagePullPolicy: IfNotPresent
```

Comment:
Why change from always?

Reply:
Slows the build. For images that we don't build, that are properly tagged and versioned in quay or docker, it seems un-necesssary to pull every time. For images we do build I'm pulling always. If you object I can revert and we can reconsider later.

Reply:
The container engine will only pull the image if the digest changes, but it has to get the digest every time the container runs, so it still has some impact. Maybe it's not worth bothering about, I should do some measurements. 
https://kubernetes.io/docs/concepts/containers/images/
```
Always
    every time the kubelet launches a container, the kubelet queries the container image registry to resolve the name to an image [digest](https://docs.docker.com/engine/reference/commandline/pull/#pull-an-image-by-digest-immutable-identifier). If the kubelet has a container image with that exact digest cached locally, the kubelet uses its cached image; otherwise, the kubelet pulls the image with the resolved digest, and uses that image to launch the container.
```